src/main/java/gregtech/api/util/FacingPos.java

```diff
@@ -27,11 +30,11 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         FacingPos facingPos = (FacingPos) o;
-        return GTUtility.arePosEqual(facingPos.pos, pos) && facing == facingPos.facing;
+        return GTUtility.arePosEqual(facingPos.getPos(), pos) && facing == facingPos.getFacing();
```

Comment:
Optional: same order in comparison
```suggestion
        return GTUtility.arePosEqual(pos, facingPos.getPos()) && facing == facingPos.getFacing();
```